### DISCUSSION TITLE
LoggingEventCompositeJsonEncoder  [ignore] message field error.

### DISCUSSION THREAD

**Discussion by Liudapeng** [logfellow/logstash-logback-encoder#485] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
 I can't ignore the message field with the encoder LoggingEventCompositeJsonEncoder. Is it because it is used incorrectly?

**Describe the solution you'd like**

**Describe alternatives you've considered** 

**Additional context** 
![image](https://user-images.githubusercontent.com/3293503/93069143-e468bb00-f6af-11ea-8d84-04c783b6a3a6.png)

**Comment by philsttr** [comment, marked as answer]
Hi @Liudapeng 

To ignore the message field when using `LoggingEventCompositeJsonEncoder`, do not configure the `message` json provider.

This example includes the `message` provider.  Therefore, the message field will be written.

```xml
<encoder class="net.logstash.logback.encoder.LoggingEventCompositeJsonEncoder">
  <providers>
    <timestamp/>
    <version/>
    <logLevel/>
    <loggerName/>
    <message/>
    <mdc/>
    <stackTrace/>
  </providers>
</encoder>
```

And this example does NOT include the message provider.  Therefore, the message field will NOT be written.

```xml
<encoder class="net.logstash.logback.encoder.LoggingEventCompositeJsonEncoder">
  <providers>
    <timestamp/>
    <version/>
    <logLevel/>
    <loggerName/>
    <mdc/>
    <stackTrace/>
  </providers>
</encoder>
```

**Comment by Liudapeng** [comment]
@philsttr Ok, got it, thank you very much
